dockerfiles/base/tests/Dockerfile

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
ok 
@TylerJewell do you have an idea of name for this new "bats" image ? if it needs to stay in `/dockerfiles/base/tests` (which is not like the other images) or move out to `/dockerfiles/<--some-name-->` but we have already a "tests" image.
I would think "bats" name is ok for me